docs/source/platform/operation-registry.md

```diff
@@ -290,6 +290,50 @@ const server = new ApolloServer({
 });
 ```
 
+## Migrating from 0.1-alpha.4
+
+#### On servers that safelist operations
+
+- Upgrade `apollo-server` to `2.6.2` and `apollo-server-plugin-operation-registry` to `0.2.0-alpha.1` (there are no breaking changes)
```

Comment:
When I first read "there are no breaking changes", I struggled to connect with how it was relevant. I think it is hanging without an action/attitude that we want users to adopt(confidence in the change/assuring that no additional updates are needed). I recommend adding a statement about why it's relevant. In this case, something like "so the upgrade can be made without additional work"